docs/src/doc/decisions-and-ADRs/adr/0009-rtl-support.stories.mdx

```diff
@@ -0,0 +1,32 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Documentation/Decisions and ADRs/ADRs/9) RTL support" />
+
+# 9) RTL support
+
+Date: 2020-07-24
+
+## Status
+
+Accepted
+
+## Context
+The components library has to support Right-to-Left (RTL) languages.
```

Comment:
```suggestion
The components library has to support text in Right-to-Left (RTL) scripts.
```
ff.

Languages are written in [scripts](https://en.wikipedia.org/wiki/Right-to-left) and those have a directionality, not the language itself.